core/src/main/scala/org/apache/spark/util/JsonProtocol.scala

```diff
@@ -469,6 +471,7 @@ private[spark] object JsonProtocol {
 
   def jobStartFromJson(json: JValue): SparkListenerJobStart = {
     val jobId = (json \ "Job ID").extract[Int]
+    val submissionTime = (json \ "Submission Time").extractOpt[Long]
```

Comment:
For backwards-compatibility, you should use `Utils.json` option here; see the block comment at https://github.com/sarutak/spark/blob/SPARK-5231/core/src/main/scala/org/apache/spark/util/JsonProtocol.scala#L46, plus the examples elsewhere in this file.